addons/test_mail/__manifest__.py

```diff
@@ -30,6 +30,9 @@
         'web.tests_assets': [
             'test_mail/static/tests/helpers/*',
         ],
+        'web.assets_tests': [
```

Comment:
Just a remark as it seems it is done that way in other modules but it is strange to have web.tests_assets and web.assets_tests.